### PR TITLE
PAYARA-3460 cleanup jdbc-core

### PR DESCRIPTION
**Pull request by Cousjava**

### CODE REVIEW

**Comment by Cousjava** [comment]
Jenkins test please

**Comment by Cousjava** [comment]
Jenkins test please

File: appserver/jdbc/jdbc-ra/jdbc-core/src/main/java/com/sun/gjc/spi/base/AbstractDataSource.java [hunk 1]
```diff
@@ -77,7 +78,7 @@
 
     private ConnectionHolder.ConnectionType conType_;
 
-    protected final static Logger _logger;
+    protected static final Logger _logger;
 
     static {
```
Comment (smillidge) [inline appserver/jdbc/jdbc-ra/jdbc-core/src/main/java/com/sun/gjc/spi/base/AbstractDataSource.java]: Any reason you've lefy this static initialiser block in?

**Comment by Cousjava** [comment]
Jenkins test please

**Pull request merged**